src/presentation/test/integration/registration/which-type.test.ts

```diff
@@ -84,4 +85,43 @@ describe("Which type Page", () => {
       }
     });
   });
+
+  it("should redirect to name page and retain ids in url if no change in type selected", async () => {
+    const limitedPartnership = new LimitedPartnershipBuilder()
+      .withId(appDevDependencies.limitedPartnershipGateway.submissionId)
+      .withPartnershipType(PartnershipType.PFLP)
+      .build();
+
+    appDevDependencies.limitedPartnershipGateway.feedLimitedPartnerships([limitedPartnership]);
+
+    const whichTypeUrl = getUrl(WHICH_TYPE_WITH_IDS_URL);
+    const nameUrl = getUrl(NAME_WITH_IDS_URL);
```

Comment:
Does this need a variable? It looks too much like the NAME_URL below.

Reply:
probably doesn't need it, i can remove it

Reply:
removed it from the other test as well